lib/components/nav-bar-component.js

```diff
@@ -28,7 +28,7 @@ export default class NavBarComponent extends AsyncBaseContainer {
 		);
 	}
 	async clickMySites() {
-		const mySitesSelector = by.css( 'header.masterbar a.masterbar__item' );
+		const mySitesSelector = by.css( '.masterbar__item-content' );
```

Comment:
@Stojdza This causes tests to fail on mobile for me. For example, try running wp-plan-purchase-spec on mobile.

If this wasn't a part of the instability maybe we could leave it as is before or maybe

```suggestion
		const mySitesSelector = by.css( '.masterbar__item[data-tip-target="my-sites"]' );
```